cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -26,19 +26,584 @@ Cask Data Application Platform Release Notes
+  
+- `CDAP-3064 <https://issues.cask.co/browse/CDAP-3064>`__ -
+  Added an ArtifactClient to interact with Artifact HTTP RESTful APIs.
+  
+- `CDAP-3283 <https://issues.cask.co/browse/CDAP-3283>`__ -
+  Added artifact information to Application RESTful APIs and the means to filter
+  applications by artifact name and version.
+  
+- `CDAP-3319 <https://issues.cask.co/browse/CDAP-3319>`__ -
+  Set the Artifact extension to be limited to one-level deep.
+  
+- `CDAP-3324 <https://issues.cask.co/browse/CDAP-3324>`__ -
+  Added a RESTful API for creating an application from an artifact.
+  
+- `CDAP-3367 <https://issues.cask.co/browse/CDAP-3367>`__ -
+  Added the ability to delete an artifact.
+  
+- `CDAP-3488 <https://issues.cask.co/browse/CDAP-3488>`__ -
+  Changed the ETLBatchTemplate from an ApplicationTemplate to an Application.
+
+- `CDAP-3535 <https://issues.cask.co/browse/CDAP-3535>`__ -
+  Added an API for programs to retrieve their application specification at runtime.
+  
+- `CDAP-3554 <https://issues.cask.co/browse/CDAP-3554>`__ -
+  Changed the plugin types from 'source' to either 'batchsource' or 'realtimesource', and
+  from 'sink' to either 'batchsink' or 'realtimesink' to reflect that the plugins
+  implement different interfaces.
+
+- `CDAP-1554 <https://issues.cask.co/browse/CDAP-1554>`__ -
+  Moved constants for default and system namespaces from Common to Id.
+
+- `CDAP-3388 <https://issues.cask.co/browse/CDAP-3388>`__ -
+  Added interfaces to ``cdap-spi`` that abstract StreamEventRecordFormat (and dependent
+  interfaces) so users can extend the ``cdap-spi`` interfaces.
+
+- `CDAP-3583 <https://issues.cask.co/browse/CDAP-3583>`__ -
+  Added a RESTful API for retrieving the metadata associated with a particular run of a
+  CDAP program.
+  
+- `CDAP-3632 <https://issues.cask.co/browse/CDAP-3632>`__ -
+  Added a RESTful API for computing lineage of a CDAP dataset or stream.
+
+Deprecated and Removed Features
+-------------------------------
+
+- See the :ref:`CDAP 3.2.0 Javadocs <javadocs>` for a list of deprecated and removed APIs.
+
+- `CDAP-2667 <https://issues.cask.co/browse/CDAP-2667>`__ -
+  Removed application templates and adapters RESTful APIs, as these templates and adapters
+  have been replaced with applications that can be controlled with the Lifecycle HTTP
+  RESTful API.
+
+- `CDAP-3596 <https://issues.cask.co/browse/CDAP-3596>`__ -
+  Replaced the ETL ApplicationTemplates with the new ETL Applications.
+
 .. _known-issues-320:
 
 Known Issues
 ------------
 
+- `CDAP-3361 <https://issues.cask.co/browse/CDAP-3361>`__ -
+  CDAP-UI should use 'name' instead of 'id' fields when listing app programs.
+  
+- `CDAP-3396 <https://issues.cask.co/browse/CDAP-3396>`__ -
+  CDAP-UI scroll bar for logs has jitters and is unbounded at the top.
+  
+- `CDAP-3409 <https://issues.cask.co/browse/CDAP-3409>`__ -
+  java.lang.NoClassDefFoundError: co/cask/cdap/data/dataset/DatasetCreationSpec
```

Comment:
lets remove this. I'm going to close this bug as won't fix. we do not guarantee compatibility across versions in standalone mode.